uvicgse/project-virtual-team-2
# Minor Improvements to UI

* Fixed Navbar layout
* Changed font size of repo path and of footer buttons
* Gave a few buttons the CSS property `cursor: pointer` allowing better affordance recognition for users
* Fixed issue where repo would sometimes be blank

### To Test
* Verify sign in/out button is inline in navbar
* Verify that font-size is reduced slightly in repo path (in navbar) and in footer buttons
